SR2021/2020-11-13-sr2021-kickstart-details.md

```diff
@@ -0,0 +1,36 @@
+---
+to: Potential team leaders + signed up teams
+subject: Student Robotics Kickstart Information
+---
+
+Hello there!
+
+We're pleased to announce some more details for the Student Robotics 2020 kickstart.
+
+As you know, following on from the success of our [previous virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/), we will be running the competition in a simulated environment this year.
+
+## Kickstart
+
+Kickstart will be streamed live on **21st November 2020 at 13:00 (UK Time)**, aiming to finish around 16:00. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.
+
+The simulator will be based in [webots](https://cyberbotics.com/#download). The download is large, and we recommend downloading it before the event to ensure you can get started immediately. Information on downloading our simulation will be made available on [the docs](https://studentrobotics.org/docs/competition-simulator/) during the event.
+
+During the event, we recommend setting up a call for you and your team to communicate and collaborate, both to share ideas and discuss potential questions. Alternatively (or as well as), you can use our Discord server (see below).
+
+After the main presentation we'll move into microgames, which are designed to help you get familiar with the simulator and robot.
+
+Visit [our website](https://studentrobotics.org/events/sr2021/kickstart/) for more information, and a link to [the livestream](https://www.youtube.com/watch?v=cQOgo0Gh4iA).
+
+## Discord
+
+If you've signed up for the competition, you'll soon receive an invite to our Discord server for your team. The invite link will be valid for `n` competitors, and should not be shared to those outside your team. If your institution has requested multiple teams, you'll receive multiple login links. We recommend sending these invites before kickstart to ensure they can get started straight away.
```

Comment:
Historically we haven't opened the forums until the day of Kickstart. Are we doing something similar with the Discord? (Is that something we _can_ do if sharing the links early?)
For clarity I'm :+1: on sharing the links ASAP, this is just something for us to bear in mind.

Reply:
It's something we could do, but it'd look very strange, and we'd want to warn them. Personally i'm ok with it being open beforehand